src/foam/u2/view/UnstyledTableView.js

```diff
@@ -253,6 +253,10 @@ foam.CLASS({
           return foam.nanos.column.ColumnConfigToPropertyConverter.create();
         return this.__context__.columnConfigToPropertyConverter;
       }
+    },
+    {
+      name: 'onDAOUpdate',
```

Comment:
Perhaps a better choice of name. Normally I would expect something named onXYZ to be a callback/listener.

Reply:
renamed